### DISCUSSION TITLE
Webcam support

### DISCUSSION THREAD

**Discussion by thunders82** [neutrinolabs/xrdp#3098] [Q&A] [answered]
Hi everyone,
It seems there is no support for webcam forwarding. It isn't easy to find information about it and I wanted to know if there is any work in progress or a timeline to implement it. The posts I found are a bit old.
It would also be good to mention it in the wiki.

Thank you

**Comment by matt335672** [comment, marked as answer]
@thunders82 - nothing has changed since I made [this comment](/neutrinolabs/xrdp/issues/2187#issuecomment-1077526794) and the one below it.

In short; this isn't supported and there are no plans to do so at the moment.

**Reply by thunders82** [reply]
Thank you for the reply @matt335672 
I was secretly hoping there would be a roadmap for it.
Thank you again for all the hard work you guys are doing with XRDP.
